packages/express/src/__tests__/acceptance/invoke-middleware.acceptance.ts

```diff
@@ -28,6 +30,30 @@ describe('Express middleware registry', () => {
     await helper.assertSpyLog();
   });
 
+  it('invokes middleware', async () => {
+    const logMiddleware = toMiddleware(spyMiddlewareFactory({action: 'log'}));
+    const mockMiddleware = toMiddleware(spyMiddlewareFactory({action: 'mock'}));
+
+    server.middleware(logMiddleware, {chain: 'log'});
+    server.middleware(mockMiddleware, {chain: 'mock'});
+
+    server.middleware(async (ctx, next) => {
+      invokeMiddleware(ctx, {
+        chain: 'log',
+        next: () => {
+          return invokeMiddleware(ctx, {chain: 'mock', next});
```

Comment:
mostly this PR is trying to allow invoking one chain after another, is it ?

Reply:
Yes, it's to expose the chaining/cascading capability.